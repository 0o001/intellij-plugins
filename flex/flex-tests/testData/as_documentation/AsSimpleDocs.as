////////////////////////////////////////////////////////////////////////////////
//
//  Copyright (C) 3000 by me.
//
////////////////////////////////////////////////////////////////////////////////

package controls
{

// !!!!
[Exclude(name="maxHorizontalScrollPosition", kind="property")]

/**
 *  The CCC control is super special control.
 * @see controls.CCC
 */
public class CCC {
  /**
   * Constructor
   */
    function CCC() {}
    function aaa() {
      a:C<caret>CC = null;
    }
  }
}
