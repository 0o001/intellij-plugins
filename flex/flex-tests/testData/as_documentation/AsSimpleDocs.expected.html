<div class='definition'><pre>CCC</pre></div><div class='content'><p>The CCC control is super special control.</div><table class='sections'><tr><td valign='top' class='section'><p>See also:</td><td><a href="psi_element://controls.CCC">controls.CCC</a></td></table>